### PR TITLE
Add .editorconfig to reduce formatting issues

### PR DESCRIPTION
**Pull request by havenchyk**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = space
+indent_size = 2
```
Comment (pleerock) [inline .editorconfig]: are you sure it completely follows our code style? I think we have indent_size set to 4 currently, right?

Comment (havenchyk) [reply .editorconfig]: I'll re-check :)

**Comment by Kononnable** [comment]
We could also think about adding `trim_trailing_whitespace` rule.

**Comment by havenchyk** [comment]
@Kononnable added `trim_trailing_whitespace`

**Comment by pleerock** [comment]
Thank you!

**Pull request merged**